puppet/modules/profiles/manifests/discourse.pp

```diff
@@ -2,8 +2,25 @@
 # @see https://github.com/discourse/discourse/blob/main/docs/INSTALL-cloud.md
 class profiles::discourse {
   $root = '/var/discourse'
-  # TODO: manage docker
-  # TODO: vcsrepo https://github.com/discourse/discourse_docker.git on $root
+
+  if $facts['os']['family'] == 'RedHat' {
+    yumrepo { 'docker-ce-stable':
+      descr   => 'Docker CE Stable - $basearch',
+      baseurl => 'https://download.docker.com/linux/centos/$releasever/$basearch/stable',
+      gpgkey  => 'https://download.docker.com/linux/centos/gpg',
+    }
+
+    ensure_packages(['docker-ce'], { require => Yumrepo['docker-ce-stable'] })
+
+    service { 'docker':
+      ensure  => 'running',
+      enable  => true,
+      require => Package['docker-ce'],
+    }
+
+    include discourse
```

Comment:
I don't see a `discourse/manifests/init.pp`

Reply:
it's because someone stupid forgot to `git add` ;)